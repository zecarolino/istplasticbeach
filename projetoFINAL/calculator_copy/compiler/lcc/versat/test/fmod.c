int _mod(int a, int b) {
	while (a >= b) a -= b;
	return a;
}
int main() { return 129 % 5; }
