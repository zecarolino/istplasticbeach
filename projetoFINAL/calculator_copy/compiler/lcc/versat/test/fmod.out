VCD info: dumpfile xtop.vcd opened for output.
4
Catched Trap at time  371 clock cycles:
